src/mmt-odk/src/info/kwarc/mmt/odk/SCSCP/Example/ClientExample.scala

```diff
@@ -9,7 +9,7 @@ object ClientExample {
     import info.kwarc.mmt.odk.OpenMath._
 
     // establish a connection
-    val client = SCSCPClient("scscp.gap-system.org")
+    val client = SCSCPClient("localhost")
```

Comment:
`scscp.gap-system.org` currently doesn't exist.

 Hardcoding any kind of a hostname (except maybe localhost) 
seems like an inherently bad idea to me.